index.js

```diff
@@ -1,131 +1,18 @@
 const readline = require('readline');
 const rl = readline.createInterface(process.stdin, process.stdout);
 
-let max = process.argv[2];
-let min = 0;
-let guessCount = 0;
-
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     rl.question(questionText, resolve);
   });
 }
 
-async function start() {
-  console.log(`   _________________________________________________________
-  //=======================================================\\
-  \\..............WELCOME TO THE ONE AND ONLY..............//
-  //.................---------------------.................\\
-  \\.................||  GUESS WHAT!!!  ||.................//
-  //.................---------------------.................\\
-  \\=======================================================//
-   <><><><><><><><><><><><><><>*<><><><><><><><><><><><><><>\n\n`);
+start();
 
+async function start() {
   console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  await numInt();
-  async function numInt() {
-    let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-    if (isNaN(parseInt(secretNumber)) || parseFloat(secretNumber) % 1 != 0) {    //Ensures user enters an integer
-      console.log("Please enter an integer...\n");
-      await numInt();
-    } else {
-      console.log('You entered: ' + secretNumber);
-      await playComp(secretNumber);
-    }
-  }
-}
-
-function guess(min, max) {
-  return Math.floor((parseInt(max)+min)/2); //Every guess is the middle of the remaining range of numbers
-}
-
-async function correctLetter (string, option1, option2) {
-  if (string.toUpperCase() === option1 || string.toUpperCase() === option2) {
-    return string.toUpperCase();
-  } else {
```

Comment:
You could take this code out of the `if` block and put it below the while loop to make your code a little more concise, and remove the outer logic check since your `while` loop is already checking the logic for you.